WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsUtils.java

```diff
@@ -798,4 +798,10 @@ public static void trackRecommendAppFetchFailed(RecommendAppSource source, Strin
 
         AnalyticsTracker.track(Stat.RECOMMEND_APP_CONTENT_FETCH_FAILED, properties);
     }
+
+    public static void trackBlockEditorEvent(String event, Map<String, Object> properties) {
+        if(event.equals("insertBlock")) {
```

Comment:
Not sure if we want to have this come over the bridge as `block_inserted` or `editor_block_inserted`. That will complicate the redux middleware in GB mobile a bit since the actions in Redux are camel case.